### ISSUE TITLE
Removing UPS integration

### ISSUE THREAD

**Issue by maartenvg** [Shopify/active_shipping#544]
Due to partnership terms with UPS we're removing the UPS integration in both the current version and past commits. To do this I will
- rewrite history to remove any occurrence of the UPS integration in a separate [branch](https://github.com/Shopify/active_shipping/tree/purge-ups-properly)

**Comment by jimbert** [comment]
Why do you plan to remove UPS integration?

**Comment by holman** [comment]
Why this change? I mean, obviously UPS is a terrible organization, but I don't see any reason why including a wrapper around their API — that you already have to sign up for — would pose problems.

Certainly you're still using these codepaths at Shopify; going back and wiping the entire project of any trace of UPS is pretty ridiculous.

**Comment by dhonig** [comment]
what is the motivation for this change?  Was there some kind of legal issue in having it remain?

**Comment by jtokoph** [comment]
Does anyone have a fork/clone with the original history?

**Comment by dhonig** [comment]
Yes. Email me at daniel@boomerdigital.net i can send you a zip tomorrow

**Comment by tblanchard** [comment]
Super annoying, why was this done?

**Comment by BenAkroyd** [comment]
I would also like to know the reason UPS requested the removal, and why the git history was rewritten to remove all trace of the code

**Comment by mikeswimm** [comment]
This is absurd. I sent an email to my rep this morning. I will share anything I learn.

**Comment by sabotatore** [comment]
@maartenvg What is the status of 2.2.0 gem version? Are you going to release it?

**Comment by jiblits** [comment]
[This might be why.](https://www.zdnet.com/article/shopify-partners-with-ups-for-negotiated-shipping-rates/)

There's still gems with the old code, so a fork or extension of active_shipping that isn't associated with Shopify would probably need to be created where this code can live.
